.github/ISSUE_TEMPLATE/config.yml

```
@@ -1 +1,17 @@
 blank_issues_enabled: false
+contact_links:
+  - name: Start a discussion
+    url:  https://github.com/FreeTubeApp/FreeTube/discussions/categories/general
+    about: Feel free to start a discussion
+  - name: Ask a question
+    url:  https://github.com/FreeTubeApp/FreeTube/discussions/categories/q-a
+    about: Ask and answer questions
+  - name: Join our Matrix Community
+    url: https://matrix.to/#/+freetube:matrix.org
+    about: "Note: Bugs and Feature requests should be made on GitHub and not in the Matrix room."
+  - name: Help translate FreeTube on Weblate
+    url: https://hosted.weblate.org/engage/free-tube/
+    about: Help translate FreeTube on Weblate
+  - name: View the FreeTube docs
```

I think this is better as we dont label it docs anywhere

```suggestion
  - name: View the FreeTube Documentation
```